SalesforceCommerceCloud/pwa-kit
# Use oid name as token key

GUS: https://gus.my.salesforce.com/a07EE00000Bgyb0YAB

# Description

When developing, it is normal for developers to change the ODS the PWA Kit is connected to. However, currently the PWA Kit will fail authentication when developers do this. This is because tokens for the previous ODS are retained in local storage and are used when connecting to the new ODS.

This change utilizes the ODS name during Auth construction so that a different ODS tokens can be stored without causing auth issues.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Set up a separate ODS instance
- Start the PWA with the default ODS settings. Verify you can auth and visit the PLP and PDP
- Check localstorage and observe the stored tokens
- Configure your PWA to use the separate ODS instance
- Restart the PWA so your configuration changes are applied
- Verify you can auth and visit the PLP and PDP (Before this fix, this will fail with an error!)
- Verify the stored tokens have changed

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
